easybuild/easyconfigs/s/SoX/SoX-14.4.2-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# Institution: IT4Innovations National Supercomputing Center, Czech Republic
+# Author: Jakub Kropacek
+# License: GPLv3
+# Year: 2022
+
+easyblock = 'ConfigureMake'
+
+name = 'SoX'
+version = '14.4.2'
+
+homepage = 'http://sox.sourceforge.net/'
+docurls = 'http://sox.sourceforge.net/Docs/Documentation'
+description = """Sound eXchange, the Swiss Army knife of audio manipulation"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b45f598643ffbd8e363ff24d61166ccec4836fea6d3888881b8df53e3bb55f6c']
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = [
+    ('FLAC', '1.3.4'),
+    ('LAME', '3.100'),
+    ('libvorbis', '1.3.7'),
+    ('FFmpeg', '5.0.1'),
```

Comment:
This should have been `FFmpeg` 4.4.2, to ensure compatibility with easyconfigs using `foss/2022a` toolchain that already depend on `FFmpeg` 4.4.2.

Fixed in #17998